server/src/main/java/com/cloud/api/query/QueryManagerImpl.java

```diff
@@ -346,14 +345,17 @@
     private DiskOfferingJoinDao _diskOfferingJoinDao;
 
     @Inject
-    private DiskOfferingDetailsDao diskOfferingDetailsDao;
+    private DiskOfferingDetailsDao _diskOfferingDetailsDao;
```

Comment:
why add the '_'? we are trying to get away from that convention.

Reply:
Consistency in the file, unless you'd like me to rename all the class variables ?

Reply:
don't see the use of this consistency. the underscore does not value in field names, does it?

Reply:
I'm for changing all variables, not just one, which can be done in a separate PR